### ISSUE TITLE
User Model

### ISSUE THREAD

**Issue by benbaka** [lcabrini/aluguntugui#15]
The spec doc for the User model is: https://github.com/lcabrini/aluguntugui/wiki/Spec:-User-Model.

The implementation is before spec: As such, this document should guide the process of fleshing out what comprises the User model.

I want to add the attributes
1. Name
2. Username
3. Password
4. date of birth

**Comment by benbaka** [comment]
It is obvious that for the Password it has to be hashed? Should we use MD5 ?

Thoughts ?

**Comment by lcabrini** [comment]
Yes, it should be hashed. I found this which may be helpful: http://stackoverflow.com/questions/11705574/hashing-a-password-and-requiring-it-for-delete-action
